XML.js

```diff
@@ -46,24 +73,17 @@ function parse (data) {
         }
       });
     } else {
-      if (hasTextNode(element))
+      if (hasTextNode(element)) {
```

Comment:
This curly braces are surrounding a if-then-else clause just like the ones above (lines 37-46). Since you used them before, I added them here also.